spec/types/calculation.md

```diff
@@ -257,8 +260,11 @@ This algorithm takes a `CalculationValue` `value` and returns a
 
   [simplifying]: #simplifying-a-calculation
 
-* Otherwise, `value` must be a `CalculationOperation`. Let `left` and `right` be
-  the result of simplifying `value.left` and `value.right`, respectively.
+* Otherwise, `value` must be a `CalculationOperation`. If within a supports
+  declaration, return `value` as-is.
+
+* Otherwise, let `left` and `right` be the result of simplifying `value.left`
+  and `value.right`, respectively.
```

Comment:
Do we ever invoke this procedure from anywhere other than "Simplifying a Calculation"? If not, you probably don't need to adjust it.